src/ck3/data/characters.rs

```diff
@@ -487,84 +469,58 @@ impl Character {
-                    }
-                }
-                AddPressedClaim => {
-                    if !claims.insert(token.clone()) {
-                        let msg = format!("{character} already has the pressed claim");
-                        let mut loc = claims.get(&token).unwrap().loc.clone();
-                        loc.column = 0;
-                        warn(ErrorKey::Validation).msg(msg).loc(token).loc(loc, "from here").push();
-                    }
-                }
-                RemoveClaim => {
-                    if !claims.remove(&token) {
-                        let msg = format!("{character} does not claim the removed claim");
-                        warn(ErrorKey::Validation).msg(msg).loc(token).push();
+                AddSpouse => {
+                    if !spouses.insert(token.clone()) {
+                        let msg = format!("{character} had already added {token} as spouse");
+                        let curr_token = spouses.get(&token).unwrap();
+                        warn(ErrorKey::History)
+                            .msg(msg)
+                            .loc(token)
+                            .loc(curr_token, "from here")
+                            .push();
                     }
                 }
-                AddSpouse => spouse = Some(token),
                 RemoveSpouse => {
-                    if let Some(current_spouse_token) = spouse {
-                        if current_spouse_token != token {
-                            let msg = format!(
-                                "{character} has a different spouse {current_spouse_token}"
-                            );
-                            let mut loc = current_spouse_token.loc.clone();
-                            loc.column = 0;
-                            warn(ErrorKey::Validation)
-                                .msg(msg)
-                                .loc(token)
-                                .loc(loc, "from here")
-                                .push();
-                        }
-                    } else {
-                        let msg = format!("{character} does not have a spouse to remove");
-                        warn(ErrorKey::Validation).msg(msg).loc(token).push();
+                    if !spouses.remove(&token) {
+                        let msg = format!("{character} did not have {token} as a spouse");
+                        warn(ErrorKey::History).msg(msg).loc(token).push();
                     }
-                    spouse = None;
                 }
                 Employed => employed = true,
+                Unemployed => employed = false,
                 GiveCouncilPosition => {
                     if !employed {
-                        let msg = format!("{character} is not currently employed");
-                        warn(ErrorKey::Validation).msg(msg).loc(token).push();
+                        let msg = format!("{character} was not employed");
```

Comment:
I think this message might be confusing because people will think having a council position means being employed. This is really about whether the character is at someone's court, right? A better message might be "{character} not at a court".

Btw many of these messages might be improved with an "at {date}", giving the date of the current history block. It more clearly tells what's the problem.